native-only.mk

```
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DOCKER_BUILD_PLATFORM_OPTIONS = --platform=linux/amd64
-
+#DOCKER_BUILD_PLATFORM_OPTIONS = --platform ?= linux/amd64
+DOCKER_BUILD_OPTIONS = --output=type=image,push=$(PUSH_ON_BUILD) --provenance=$(ATTACH_ATTESTATIONS) --sbom=$(ATTACH_ATTESTATIONS)
```

What is the rationale behind this change?

---

So I commented out the DOCKER_BUILD_PLATFORM_OPTIONS because we were already specifying it in the workflow in ci.yaml. I added the DOCKER_BUILD_OPTIONS to store the image so we can build the Manifest in the next job.

---

Instead of removing DOCKER_BUILD_PLATFORM_OPTIONS, you can use the `?=` operator. `?=` indicates that the variable is set to a value only if the variable is empty or unset
```suggestion
DOCKER_BUILD_PLATFORM_OPTIONS ?= --platform=linux/amd64
DOCKER_BUILD_OPTIONS = --output=type=image,push=$(PUSH_ON_BUILD) --provenance=$(ATTACH_ATTESTATIONS) --sbom=$(ATTACH_ATTESTATIONS)
```